package.json

```diff
@@ -9,7 +9,8 @@
     "fmt:check": "biome check . --diagnostic-level=error && prettier --check .",
     "fmt": "biome check . --write --diagnostic-level=error --organize-imports-enabled=true && prettier --write .",
     "verify-redirects": "node snippets/custom-scripts/check-redirect-conflicts.js",
-    "test-links": "mint broken-links"
+    "test-links": "mint broken-links",
+    "vale": "vale --glob='!{snippets,errors}/**' ."
```

Comment:
Is it possible to add vale to the dependencies? When I try to run this command I get the error that vale is not installed

Also, we should mention this command in the readme